csharp/private/frameworks/net60.BUILD

```diff
@@ -0,0 +1,1442 @@
+        "//:System.Reflection.TypeExtensions",
+        "//:System.Resources.Reader",
+        "//:System.Resources.ResourceManager",
+        "//:System.Resources.Writer",
+        "//:System.Runtime",
+        "//:System.Runtime.CompilerServices.Unsafe",
+        "//:System.Runtime.CompilerServices.VisualC",
+        "//:System.Runtime.Extensions",
+        "//:System.Runtime.Handles",
+        "//:System.Runtime.InteropServices",
+        "//:System.Runtime.InteropServices.RuntimeInformation",
+        "//:System.Runtime.Intrinsics",
+        "//:System.Runtime.Loader",
+        "//:System.Runtime.Numerics",
+        "//:System.Runtime.Serialization",
+        "//:System.Runtime.Serialization.Formatters",
+        "//:System.Runtime.Serialization.Json",
+        "//:System.Runtime.Serialization.Primitives",
+        "//:System.Runtime.Serialization.Xml",
+        "//:System.Security",
+        "//:System.Security.AccessControl",
+        "//:System.Security.Claims",
+        "//:System.Security.Cryptography.Algorithms",
+        "//:System.Security.Cryptography.Cng",
+        "//:System.Security.Cryptography.Csp",
+        "//:System.Security.Cryptography.Encoding",
+        "//:System.Security.Cryptography.OpenSsl",
+        "//:System.Security.Cryptography.Primitives",
+        "//:System.Security.Cryptography.X509Certificates",
+        "//:System.Security.Principal",
+        "//:System.Security.Principal.Windows",
+        "//:System.Security.SecureString",
+        "//:System.ServiceModel.Web",
+        "//:System.ServiceProcess",
+        "//:System.Text.Encoding",
+        "//:System.Text.Encoding.CodePages",
+        "//:System.Text.Encoding.Extensions",
+        "//:System.Text.Encodings.Web",
+        "//:System.Text.Json",
+        "//:System.Text.RegularExpressions",
+        "//:System.Threading",
+        "//:System.Threading.Channels",
+        "//:System.Threading.Overlapped",
+        "//:System.Threading.Tasks",
+        "//:System.Threading.Tasks.Dataflow",
+        "//:System.Threading.Tasks.Extensions",
+        "//:System.Threading.Tasks.Parallel",
+        "//:System.Threading.Thread",
+        "//:System.Threading.ThreadPool",
+        "//:System.Threading.Timer",
+        "//:System.Transactions",
+        "//:System.Transactions.Local",
+        "//:System.ValueTuple",
+        "//:System.Web",
+        "//:System.Web.HttpUtility",
+        "//:System.Windows",
+        "//:System.Xml",
+        "//:System.Xml.Linq",
+        "//:System.Xml.ReaderWriter",
+        "//:System.Xml.Serialization",
+        "//:System.Xml.XDocument",
+        "//:System.Xml.XmlDocument",
+        "//:System.Xml.XmlSerializer",
+        "//:System.Xml.XPath",
+        "//:System.Xml.XPath.XDocument",
+        "//:WindowsBase",
+    ],
+    visibility = ["@net//:__pkg__"],
+)
+
+import_library(
+    name = "Microsoft.CSharp",
+    target_framework = "net6.0",
+    refdll = ":ref/net6.0/Microsoft.CSharp.dll",
+    deps = [":mscorlib"],
```

Comment:
Generator spit out mscorlib for everything. Don't recall if we manually changed that for previous frameworks.